.eslintrc.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
     },
     overrides: [
         {
-            files: ['js/ts/**/*.ts'],
+            files: ['js/ts/**/*.ts', 'playwright/**/**/*.ts'],
```

Comment:
```suggestion
            files: ['js/ts/**/*.ts', 'playwright/**/*.ts'],
```

I think the `/**/` glob pattern targets directories any level deep, so the doubling up of it shouldn't be needed here?

Reply:
Ah, yup! Just tested to make sure. I'll get these fixed up.